scripts/aggregate-translations/write-i18n-loaders.js

```diff
@@ -2,25 +2,50 @@ const path = require('path');
 const startCase = require('lodash.startcase');
 const supportedLocales = require('./i18nSupportedLocales');
 
-const createIntlLoader = (loaderName, locale) => (
-  `var ${loaderName} = function ${loaderName}() {
+const createIntlLoader = (loaderName, locale, format) => {
+  if (format === 'modern') {
+    return (
+      `const ${loaderName} = () =>
+   import('intl/locale-data/jsonp/${locale}.js')
+   .catch(error => console.log('An error occurred while loading intl/locale-data/jsonp/${locale}.js'));\n
```

Comment:
Lets throw add the error to the error message

Reply:
added